libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestClient.java

```diff
@@ -233,7 +233,7 @@ public String getRefreshToken() {
 	 * @return The client info.
 	 */
 	public ClientInfo getClientInfo() {
-		return clientInfo;
+		return oAuthRefreshInterceptor.clientInfo;
```

Comment:
Maybe we should remove `clientInfo` instance variable from this class and use `oAuthRefreshInterceptor.clientInfo` everywhere? I don't see any reason to keep the instance variable around. It makes it harder to keep the two copies in sync.